src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const midTerm = 3;
+  const longDiscount = 50;
+  const midDiscount = 20;
+  const price = 40 * days;
+
+  if (days >= longTerm) {
+    return price - longDiscount;
+  }
+
+  if (days >= midTerm) {
```

Comment:
The value of the 'midTerm' variable (3) is hardcoded. It's better to make it a parameter of the function to increase its flexibility and reusability.